hawk/app/assets/stylesheets/shared/_foot.scss

```diff
@@ -2,19 +2,26 @@
 // See COPYING for license.
 
 footer {
-  background-color: #fff;
-  padding: 20px 10px;
-  border-top: 1px solid #999;
+  display: none;
```

Comment:
Properties should be ordered background-color, border-top, bottom, display, height, overflow, padding, position, width, z-index